tests/system/action/poll/test_update.py

```diff
@@ -130,3 +131,231 @@ def test_not_allowed_for_non_analog(self) -> None:
+                title="test_title_beeFaihuNae1vej2ai8m",
+                pollmethod="Y",
+                type=Poll.TYPE_NAMED,
+                onehundred_percent_base="Y",
+                majority_method="simple",
+                state=Poll.STATE_CREATED,
+                meeting_id=113,
+                option_ids=[1, 2],
+                entitled_group_ids=[1],
+            ),
+        )
+        self.create_model("option/1", {"meeting_id": 113, "poll_id": 1})
+        self.create_model("option/2", {"meeting_id": 113, "poll_id": 1})
+        self.update_model(
+            "user/1",
+            {
+                "is_present_in_meeting_ids": [113],
+                "group_$113_ids": [1],
+                "group_$_ids": ["113"],
+            },
+        )
+
+    def test_patch_title(self) -> None:
+        response = self.request(
+            "poll.update",
+            {"title": "test_title_Aishohh1ohd0aiSut7gi", "id": 1},
+        )
+        self.assert_status_code(response, 200)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("title"), "test_title_Aishohh1ohd0aiSut7gi")
+
+    def test_prevent_patching_assignment(self) -> None:
+        self.create_model(
+            "assignment/2", dict(title="test_title_phohdah8quukooHeetuz", open_posts=1)
+        )
+        response = self.request(
+            "poll.update",
+            {"content_object_id": "assignment/2", "id": 1},
+        )
+        self.assert_status_code(response, 400)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("content_object_id"), "assignment/1")  # unchanged
+
+    def test_patch_pollmethod(self) -> None:
+        response = self.request(
+            "poll.update",
+            {"pollmethod": "YNA", "id": 1},
+        )
+        self.assert_status_code(response, 200)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("pollmethod"), "YNA")
+        self.assertEqual(poll.get("onehundred_percent_base"), "YNA")
+
+    def test_patch_invalid_pollmethod(self) -> None:
+        response = self.request(
+            "poll.update",
+            {"pollmethod": "invalid"},
+        )
+        self.assert_status_code(response, 400)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("pollmethod"), "Y")
+
+    def test_patch_type(self) -> None:
+        response = self.request("poll.update", {"type": "analog", "id": 1})
+        self.assert_status_code(response, 200)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("type"), "analog")
+
+    def test_patch_invalid_type(self) -> None:
+        response = self.request("poll.update", {"type": "invalid", "id": 1})
+        self.assert_status_code(response, 400)
+        poll = self.get_model("poll/1")
+        self.assertEqual(poll.get("type"), "named")
+
+    def test_patch_not_allowed_type(self) -> None:
+        # self.update_model("organisation/1", {"enable_electronic_voting": False})
```

Comment:
you can explicitly check this.

Reply:
But check also the case with setting this to `true` and change e.g. to anonymous type.